scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -198,7 +240,7 @@ def _get_caseroot2(self):
         casename1 = self._case1.get_value("CASE")
         caseroot1 = self._case1.get_value("CASEROOT")
 
-        casename2 = "{}.{}".format(casename1, self._run_two_suffix)
+        casename2 = "{}.{}".format(casename1, self._run_two_suffix) if self._separate_runs else casename1
```

Comment:
I find this long one-liner hard to read. Please separate it into multiple lines - i.e., a standard 'if' construct.